test/controllers/declaration/DeclarationAdditionalActorsControllerSpec.scala

```diff
@@ -302,16 +304,20 @@ class DeclarationAdditionalActorsControllerSpec
     }
   }
 
-  private def withCache(data: DeclarationAdditionalActorsData) =
-    withNewCaching(aDeclaration(withChoice("SMP"), withDeclarationAdditionalActors(data)))
+  private def withCache(data: DeclarationAdditionalActorsData) = {
```

Comment:
I am confused about this method. How does the name matches what it's doing? It would be great if you could figure out some better name for it.